src/GitLurker.UI/ViewModels/FolderViewModel.cs

```diff
@@ -25,6 +27,8 @@ public FolderViewModel(string folder, Action<string> deleteCallback)
 
     public string Folder => _folder;
 
+    public Brush Foreground => Directory.Exists(Folder) ? Brushes.Black : Brushes.Red;
```

Comment:
I would just expose a `FolderExist` boolean property in the ViewModel and let the `view` decide how to display.

In the view I would do a `DataTrigger` to set the proper style.

Here's a simple example:
https://github.com/C1rdec/Git-Lurker/blob/25a6940559f497d3edb118a6eeb3fe611d00a915/src/GitLurker.UI/Views/ActionView.xaml#L25-L27

Reply:
I've implemented the change. I agree it's better this way.